src/utils.js

```diff
@@ -324,6 +324,70 @@ go.utils = {
     },
 
 
+// STAGE-BASE HELPERS
+
+    read_subscription: function(im, subscription_id) {
```

Comment:
I was trying to move to the CRUD R which is 'read' when doing api get calls.  Should I not do that?